docs/sql-programming-guide.md

```diff
@@ -1253,7 +1253,12 @@ This command builds a new assembly jar that includes Hive. Note that this Hive a
 on all of the worker nodes, as they will need access to the Hive serialization and deserialization libraries
 (SerDes) in order to access data stored in Hive.
 
-Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`.
+Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`. Please note when running 
```

Comment:
do you mean adding hive-site.xml through spark.driver.extraClassPath? when running on yarn-cluster mode, the AM node which runs the driver program can be any node on the cluster, and that node may not have Spark client installed.